code/pagetypes/Forum.php

```diff
@@ -731,9 +731,13 @@ function BBTags() {
 	 *
 	 * @return Form Returns the post message form
 	 */
-	function PostMessageForm($addMode = false, $post = false) {
-		$thread = false;
+	function PostMessageForm($addMode = true, $post = false) {
```

Comment:
I'd be against this change even in 0.8 because it's unnecessary to fix the bug and adds an upgrade friction point for no real benefit